src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -239,7 +246,7 @@ public void OutputObservedDataLinkingChanged(bool isLinkedMappedOutputs)
          {
             var outputColumnUsed = dataColumnDTO.Used;
             var linkedObservedData = getLinkedObservedDataFromOutputPath(dataColumnDTO.DataColumn.PathAsString);
-            SetUsedState(linkedObservedData, outputColumnUsed);
+            SetUsedState(linkedObservedData, outputColumnUsed, true);
```

Comment:
same. let's name the parameter instead of passing true or false. Just good for whomever looks at this in 3 moths